pkg/dns/util/util.go

```diff
@@ -119,8 +170,22 @@ func IsServiceIPSet(service *corev1.Service) bool {
 
 // GetClusterIPs returns IPs set for the service
 func GetClusterIPs(service *corev1.Service) []string {
+	clusterIPs := []string{service.Spec.ClusterIP}
```

Comment:
can you add coment why did you added this line?

Reply:
I'd add the comment but I couldn't think of any useful one here.

We prefer spec.ClusterIPs over spec.ClusterIP for dual-stack support.

ClusterIPs field is a newer field and is sort of an extension for a spec.ClusterIP field (e.g. in dual-stack clusters)
And this info is already in these fields comments.